test/regression/test_deploy.py

```diff
@@ -10,12 +10,17 @@
 from utils import Logger, StopWatch
 
 
+def get_expected_cortx_control_pods(namespace):
+    cmd = f'helm get values cortx -n {namespace} | yq .control.replicaCount'
```

Comment:
Same comment as above. You could also go native Python and use JSON (or YAML).

```python
>>> with subprocess.Popen(["helm", "get", "values", "cortx", "--output", "json"], stdout=subprocess.PIPE) as proc:
...   print(json.loads(proc.stdout.read())["control"]["replicaCount"])
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
KeyError: 'replicaCount'
>>> with subprocess.Popen(["helm", "get", "values", "cortx", "--all", "--output", "json"], stdout=subprocess.PIPE) as proc:
...   print(json.loads(proc.stdout.read())["control"]["replicaCount"])
...
1
```

Reply:
In the end I kept this the same (but fixing the problem you point out).  Something wasn't quite working (between getting json from stdout and indexing into it), and the test cycle was kind of painful.  I went with the simpler working solution.